UI/CUIControls.cpp

```diff
@@ -1924,3 +1925,136 @@ void RotatingGraphic::Render() {
     glPopClientAttrib();
     glPopMatrix();
 }
+
+////////////////////////////////////////////////
+// ToggleButton
+////////////////////////////////////////////////
+ToggleButton::ToggleButton (GG::SubTexture icon_, GG::Clr color, GG::Clr checked_clr, const std::string& tooltip_str,
```

Comment:
I agree with @geoffthemedio  here.  If you want some state use a StateButton with a new Representer.